library/src/main/java/com/google/android/exoplayer2/extractor/flv/FlvExtractor.java

```diff
@@ -288,7 +303,24 @@ public long getDurationUs() {
 
   @Override
   public long getPosition(long timeUs) {
+
+
+    double timeMs = timeUs / 1000000;
```

Comment:
There are only 1000 microseconds in a millisecond, so either the conversion or the variable naming is wrong here? I suspect the naming (i.e. timeMs should be timeSecs)?